subsys/bluetooth/host/conn.c

```diff
@@ -3160,7 +3160,7 @@ int bt_conn_auth_cb_overlay(struct bt_conn *conn, const struct bt_conn_auth_cb *
 	/* The cancel callback must always be provided if the app provides
```

Comment:
Please split `bt_conn_auth_cb_overlayfunction` in the commit message into 2 words.